chapters/ch37.md

```diff
@@ -1,71 +1,70 @@
 > # 37. How to decide whether to use all your data
 
---> _replace THIS LINE by your translation for the above line_
+# 37. Làm sao để quyết định có nên sử dụng toàn bộ dữ liệu của bạn hay không?
 
 > Suppose your cat detector’s training set includes 10,000 user-uploaded images. This data comes from the same distribution as a separate dev/test set, and represents the distribution you care about doing well on. You also have an additional 20,000 images downloaded from the internet. Should you provide all 20,000+10,000=30,000 images to your learning algorithm as its training set, or discard the 20,000 internet images for fear of it biasing your learning algorithm?
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử tập huấn luyện của bộ nhận diện mèo của bạn bao gồm 10.000 hình ảnh do người dùng tải lên. Dữ liệu này đến từ cùng một phân phối chia ra một tập phát triển/kiểm tra riêng biệt, và đại diện cho phân phối mà bạn muốn làm tốt trên đó. Bạn cũng có thêm 20.000 hình ảnh được tải xuống từ internet. Bạn có nên cung cấp tất cả 20.000 + 10.000 = 30.000 ảnh cho thuật toán learning dưới dạng tập huấn luyện, hay nên loại bỏ 20.000 ảnh từ internet để tránh thiên vị thuật toán learning của bạn?
```

Comment:
Bạn review lại các thuật ngữ đã có ở bảng glossary nhé: [glossary.md](https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/blob/master/glossary.md). Mình thấy có nhiều từ đã có mà bạn không áp dụng khi dịch:
- learning algorithm -> thuật toán học
- bias -> độ chệch
Tất nhiên bạn nên có điều chỉnh để câu văn khi dịch xuôi và dễ đọc hơn đối với từng ngữ cảnh. Bạn review lại toàn bộ trc khi mọi người review lần tiếp nhé.

Reply:
Mình đã bỏ dấu tick ở checklist trên đầu nhé. Bạn update lại và check lại để mọi người tiếp tục review nhé.

Reply:
ok @sonvx , mình đã update "learning algorithm -> thuật toán học" nha. 
Còn chỗ "fear of it biasing your learning algorithm" thì mình cũng phân vân nên dùng độ chệch nhưng sau nghĩ thì mình không cần danh từ ở đây mà cần động từ. Mình dịch lại trong commit mới nhất là "để tránh làm chệch đi thuật toán học của bạn". Bạn review lại thử nhé.